ansible/templates/minigraph_link_meta.j2

```diff
@@ -1,6 +1,25 @@
-{% if 'dualtor' in topo %}
+{% set link_defined = False %}
+
+{% if 'dualtor' in topo or
+      (macsec_card is defined and macsec_card == True and 't2' in topo) %}
+    {% set link_defined = True %}
```

Comment:
@vdahiya12 what does link_defined mean for dualtor?

Reply:
it's  a variable for link_metadata tags need to be present.